### ISSUE TITLE
UIQM  is too large

### ISSUE THREAD

**Issue by wzy0903** [xahidbuffon/funie-gan#4]
I use the measure.py to calculate the UIQM ,but the value is 300~500. In your paper,I see the valu is 1~3. I'd appreciate it if you could give me an answer

**Comment by xahidbuffon** [comment]
Maybe some quantity is blowing up to infinity, can u trace down the value and add some regularizers? 
(Note that the uiqm values in the papers are averages over 1K images, excluding a few numerically unstable outlier examples)

**Comment by wzy0903** [comment]
Is the thousand pictures paired? I used A and its groundtruth in the data folder.Are there any other pairs of test sets besides the paired training set in edvp? Or just pull 1000 from the paired training set of edvp to test and measure.

**Comment by xahidbuffon** [comment]
Hi, both paired and unpaired images were tested separately (the test set included images from euvp, data provided by SOTA models, and also arbitrary images from web). Only a few images are provided in this repository for testing - please do not stick to these 30ish images for testing.

**Comment by wzy0903** [comment]
Thank you! I will try again.
